http_logs/test_procedures/default.json

```diff
@@ -59,76 +59,50 @@
-          ,"target-throughput": {{ target_throughput | tojson }}
-          {%- endif %}
-          {%-if search_clients is defined and search_clients %}
-          ,"clients": {{ search_clients | tojson}}
-          {%- endif %}
+          "iterations": 100,
+          "target-throughput": {{ term_target_throughput or target_throughput | default(50) | tojson }},
+          "clients": {{ term_search_clients or search_clients | default(1) }}
         },
         {
           "operation": "range",
           "warmup-iterations": 100,
-          "iterations": 100
-          {%- if not target_throughput %}
-          ,"target-throughput": 1
-          {%- elif target_throughput is string and target_throughput.lower() == 'none' %}
-          {%- else %}
-          ,"target-throughput": {{ target_throughput | tojson }}
-          {%- endif %}
-          {%-if search_clients is defined and search_clients %}
-          ,"clients": {{ search_clients | tojson}}
-          {%- endif %}
+          "iterations": 100,
+          "target-throughput": {{ range_target_throughput or target_throughput | default(1) | tojson }},
+          "clients": {{ range_search_clients or search_clients | default(1) }}
         },
         {
+          "name": "status-200s-in-range",
           "operation": "200s-in-range",
           "warmup-iterations": 500,
-          "iterations": 100
-          {%- if not target_throughput %}
-          ,"target-throughput": 33
-          {%- elif target_throughput is string and target_throughput.lower() == 'none' %}
-          {%- else %}
-          ,"target-throughput": {{ target_throughput | tojson }}
-          {%- endif %}
-          {%-if search_clients is defined and search_clients %}
-          ,"clients": {{ search_clients | tojson}}
-          {%- endif %}
+          "iterations": 100,
+          "target-throughput": {{ status_200s_in_range_target_throughput or target_throughput | default(33) | tojson }},
+          "clients": {{ status_200s_in_range_search_clients or search_clients | default(1) }}
         },
         {
+          "name": "status-400s-in-range",
           "operation": "400s-in-range",
           "warmup-iterations": 500,
-          "iterations": 100
-          {%- if not target_throughput %}
-          ,"target-throughput": 50
-          {%- elif target_throughput is string and target_throughput.lower() == 'none' %}
-          {%- else %}
-          ,"target-throughput": {{ target_throughput | tojson }}
-          {%- endif %}
-          {%-if search_clients is defined and search_clients %}
-          ,"clients": {{ search_clients | tojson}}
-          {%- endif %}
+          "iterations": 100,
+          "target-throughput": {{ status_400s_in_range_target_throughput or target_throughput | default(50) | tojson }},
+          "clients": {{ status_400s_in_range_search_clients or search_clients | default(1) }}
         },
         {
           "operation": "hourly_agg",
+          "warmup-iterations": 50,
```

Comment:
I think this was changed from `100` to `50` in the process of this cherry-pick ([Reference](https://github.com/opensearch-project/opensearch-benchmark-workloads/blob/7/http_logs/test_procedures/default.json#L132)). Is there a reason why it was changed?

Reply:
No, may have been a bug in my script or the cherry-pick. I can put out a quick fix!